library/elliptics.h

```diff
@@ -447,16 +447,20 @@ void dnet_io_cleanup(struct dnet_node *n);
 
 void dnet_io_req_free(struct dnet_io_req *r);
 
+struct dnet_grpc_server_config;
 struct dnet_config_data {
 	int cfg_addr_num;
 	struct dnet_addr *cfg_addrs;
 
+	struct dnet_grpc_server_config *cfg_grpc;
```

Comment:
I found previous step: https://github.com/interiorem/elliptics/pull/245#discussion_r266077790. So it looks like now it could be moved to `config_data`, so it could be wrapped in `std::unique_ptr` for example.